services/core-web/src/components/Forms/EditMinespaceUser.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field, reduxForm } from "redux-form";
+import { Form } from "@ant-design/compatible";
+import "@ant-design/compatible/assets/index.css";
+import { Button, Col, Row } from "antd";
+import { required, requiredList } from "@common/utils/Validate";
+import { resetForm } from "@common/utils/helpers";
+import RenderField from "@/components/common/RenderField";
+import * as FORM from "@/constants/forms";
+import { renderConfig } from "@/components/common/config";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  mines: CustomPropTypes.options.isRequired,
+  // eslint-disable-next-line react/no-unused-prop-types
+  minespaceUserEmailHash: PropTypes.objectOf(PropTypes.any).isRequired,
+  handleSubmit: PropTypes.func.isRequired,
```

Comment:
Once the validation is removed, this prop is no longer needed. 'minespaceUserEmailHash', please remove, and remove from being passed into the 'OpenModal' function